src/core/services/gesture/gesture.js

```diff
@@ -248,7 +248,7 @@ function MdGesture($$MdGestureHandler, $$rAF, $timeout) {
         if (touchActionProperty) {
           // We check for horizontal to be false, because otherwise we would overwrite the default opts.
           this.oldTouchAction = element[0].style[touchActionProperty];
-          element[0].style[touchActionProperty] = options.horizontal === false ? 'pan-y' : 'pan-x';
+          element[0].style[touchActionProperty] = options.horizontal ? 'pan-y' : 'pan-x';
```

Comment:
I think I initially wrote that lines and I think it was intentional to only allow `pan-x` for horizontal gestures and `pan-y` for vertical gestures.

Can you explain more why this needs to be the other way around?

| State | Value | Description |
| ----- | ------ | --- |
| horizontal | pan-x | Only recognize X-Axis movement s |
| vertical |  pan-y | Only recognize Y-Axis movements |